Script/menu.js

```diff
@@ -6,3 +6,30 @@ const menuBtn = document.querySelector('#menuBtn');
 menuBtn.addEventListener('click', collapsible);
 const xBtn = document.querySelector('#xBtn');
 xBtn.addEventListener('click', collapsible);
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply ummy text of the printing and typesetting industry. Lorem Ipsum has been the standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    image: 'images/img-placeholder.svg',
+    featureList: ['html', 'bootstrap', 'Ruby'],
+    seeLive: 'https://alanluqman.github.io/Personal-Portfolio/',
+    seeSource: 'https://github.com/alanluqman/Personal-Portfolio',
+  }
+];
+const popUpWindow = document.querySelector('.popup-window');
```

Comment:
thanks for your review dear, but actually it works from my computer and i took a screen record for your reference . but i did not merged this branch to main so it will not work if you run the main branch.

https://www.loom.com/share/d360953195af44da9294d61d20ff4b04

Reply:
@alanluqman, I tested from the PopUp window branch on Chrome and Firefox and it didn't work on my computer. I kindly suggest that you submit it again for another review, Maybe it will work on the next reviewer's computer

Reply:
> thanks for your review dear, but actually it works from my computer and i took a screen record for your reference . but i did not merged this branch to main so it will not work if you run the main branch.
> 
> https://www.loom.com/share/d360953195af44da9294d61d20ff4b04
Edit: I'm really sorry for the requested changes. It turns out the fault is from my side. Please be patient. I will  request your project to be approved now.